libs/styles/styles.css

```diff
@@ -1,134 +1,587 @@
+  --type-heading-xxl-lh: var(--type-heading-lh, 45px);
+  --type-heading-xl-size: calc(var(--font-size-multiplier, 1) * var(--s2-font-size-900, 28px)); 
+  --type-heading-xl-lh: var(--type-heading-lh, 35px);
+  --type-heading-l-size: calc(var(--font-size-multiplier, 1) * var(--s2-font-size-800, 24px)); 
+  --type-heading-l-lh: var(--type-heading-lh, 30px);
+  --type-heading-m-size: calc(var(--font-size-multiplier, 1) * var(--s2-font-size-700, 20px)); 
+  --type-heading-m-lh: var(--type-heading-lh, 25px);
+  --type-heading-s-size: calc(var(--font-size-multiplier, 1) * var(--s2-font-size-600, 18px)); 
+  --type-heading-s-lh: var(--type-heading-lh, 22.5px);
+  --type-heading-xs-size: calc(var(--font-size-multiplier, 1) * 16px); /* original not in S2 */
   --type-heading-xs-lh: 20px;
-  --type-heading-xxs-size: calc(var(--font-size-multiplier, 1) * 14px);
+  --type-heading-xxs-size: calc(var(--font-size-multiplier, 1) * 14px); /* original not in S2 */
   --type-heading-xxs-lh: 18px;
 
+  @media (min-width: 768px) { /* Tablet */
+    --type-heading-xxl-size: var(--s2-font-size-1300);
+    --type-heading-xl-size: var(--s2-font-size-1000);
+    --type-heading-l-size: var(--s2-font-size-900);
+    --type-heading-m-size: var(--s2-font-size-800);
+    --type-heading-s-size: var(--s2-font-size-700);
+  }
+  
+  @media (min-width: 1280px) { /* Small desktop */
+    --type-heading-xxl-size: var(--s2-font-size-1300);
+    --type-heading-xl-size: var(--s2-font-size-1200);
+    --type-heading-l-size: var(--s2-font-size-1100);
+    --type-heading-m-size: var(--s2-font-size-1000);
+    --type-heading-s-size: var(--s2-font-size-900);
+  }
+  
+  @media (min-width: 1680px) { /* Large desktop */
+    --type-heading-xxl-size: var(--s2-font-size-1500);
+    --type-heading-xl-size: var(--s2-font-size-1400);
+    --type-heading-l-size: var(--s2-font-size-1200);
+    --type-heading-m-size: var(--s2-font-size-1000);
+    --type-heading-s-size: var(--s2-font-size-900);
+  }
+
   /* Headings (General) */
   --type-heading-base-lh: 1.2;
 
-  /* Body (Consonant) */
-  --type-body-xxl-size: calc(var(--font-size-multiplier, 1) * 28px);
-  --type-body-xxl-lh: 42px;
-  --type-body-xl-size: calc(var(--font-size-multiplier, 1) * 22px);
-  --type-body-xl-lh: 33px;
-  --type-body-l-size: calc(var(--font-size-multiplier, 1) * 20px);
-  --type-body-l-lh: 30px;
-  --type-body-m-size: calc(var(--font-size-multiplier, 1) * 18px);
-  --type-body-m-lh: 27px;
-  --type-body-s-size: calc(var(--font-size-multiplier, 1) * 16px);
-  --type-body-s-lh: 24px;
-  --type-body-xs-size: calc(var(--font-size-multiplier, 1) * 14px);
-  --type-body-xs-lh: 21px;
-  --type-body-xxs-size: calc(var(--font-size-multiplier, 1) * 12px);
-  --type-body-xxs-lh: 18px;
-
-  /* Detail */
-  --type-detail-all-weight: 700;
+  /* S2 Titles (new) */
+  --type-title-xl-weight: var(--s2-extra-bold-font-weight);
+  --type-title-all-weight: var(--s2-bold-font-weight);
+  --type-title-lh: 130%;
```

Comment:
I see the line height is used as percentage (e.g. `--type-title-lh`), fixed value (e.g. `--type-heading-base-lh`) and px value (e.g. `--type-heading-xxxl-lh`). Should we go the extra mile and make them consistent?

Reply:
Yea, this is because the new S2 properties are used in percentages and the Milo originals are px and fixed values there. Wasn't sure if we should change the Milo originals but I can update these for consistency.

Reply:
I'd also suggest to have these as factors, not percentages, `1.3` as opposed to `130%`